docs/learn/old/prompts.md

```diff
@@ -0,0 +1,219 @@
+# Prompts
+
+When working with LLMs, a surprising amount of work can go into determining the final text which gets passed into the model. Let's take a look at some of the tools Mirascope provides to make formatting your prompts as seamless as possible.
+
+## BasePrompt
+
+`BasePrompt` is a base class which works across all model providers. To create a prompt using `BasePrompt`:
+
+- Define a class which inherits `BasePrompt`
+- Set the prompt via the docstring and denote any template variable with curly braces
```

Comment:
If all cookbooks and documentation uses `prompt_template` decorator, would it make more sense to mention using the decorator first?